tests/performance/apdex-score/specs/test.spec.js

```diff
@@ -0,0 +1,79 @@
+const loadSettings = require('../settings-provider');
+
+const loadPage = require('@page-objects/apdex/load.page');
+const loginPage = require('@page-objects/apdex/login.page');
+const contactsPage = require('@page-objects/apdex/contacts.page');
+const targetsPage = require('@page-objects/apdex/targets.page');
+const tasksPage = require('@page-objects/apdex/tasks.page');
+const messagesPage = require('@page-objects/apdex/messages.page');
+const reportsPage = require('@page-objects/apdex/reports.page');
+
+describe('Apdex Performance Workflows', () => {
+  const settingsProvider = loadSettings();
+  const repetitions = settingsProvider.getIterations();
+  const skip = settingsProvider.getTestsToSkip();
+
+  before(async () => {
+    if (!skip.login) {
```

Comment:
This is especially useful when used with `capabilities.noReset` settings, it speeds up the test a lot, especially when we are building the tests for a new instance. Because we don't need to wait to install the app, download 7000 docs, login, etc. So a faster start.